src/utils/juror-draft-utils.js

```diff
@@ -33,7 +33,7 @@ export function transformJurorDataAttributes(jurorDraft) {
 
   return {
     ...jurorDraft,
-    weight: bigNum(weight),
```

Comment:
Just sanity checking—are we sure in these cases where we're not using a BN anymore?

Reply:
For regulars rounds the juror weight will be at most 81 (and that is if a juror get's all available slots of a round ,highly unlikely). For final rounds it will be the number of times the juror has the min active balance which it's also unlikely to be too big for it to not be supported.

Reply:
So now I'm wondering... I think we did this in a few other places to make the code simpler, but maybe we should align all the internal numbers to be BNs?

Otherwise it might get confusing later—`weight` is a normal number, but `round.number` is a string and `round.collectedTokens` is a BN. The string / BN difference can make sense (some are identifiers, some are meant for calculations), but juggling two number representations can be frustrating unless they're clearly delinated (e.g. suffixing `Bn` in variable names or [moving all the normal numbers to a separate field](https://github.com/aragon/aragon-apps/blob/master/apps/voting/app/src/app-state-reducer.js#L54)).